src/watcher/cmd/main.go

```diff
@@ -27,20 +28,25 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	var watchedNamespaces []string
+	var otterizeClientID string
+	var otterizeClientSecret string
 
-	pflag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
-	pflag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	pflag.StringVar(&metricsAddr, "metrics-bind-address", ":7070", "The address the metric endpoint binds to.")
+	pflag.StringVar(&probeAddr, "health-probe-bind-address", ":7071", "The address the probe endpoint binds to.")
 	pflag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	pflag.StringSliceVar(&watchedNamespaces, "watched-namespaces", nil,
 		"Namespaces that will be watched by the operator. Specify multiple values by specifying multiple times or separate with commas.")
+	pflag.StringVar(&otterizeClientID, "client-id", "", "Otterize cloud Kubernetes integration client id")
+	pflag.StringVar(&otterizeClientSecret, "client-secret", "", "Otterize cloud Kubernetes integration client secret")
 
 	pflag.Parse()
 
 	ctrl.SetLogger(logrusr.New(logrus.StandardLogger()))
 
 	var err error
+	cloudAddr := os.Getenv("OTTERIZE_API_ADDRESS") // we use Getenv because empty string is viable
```

Comment:
I thought about having this as an env var with a default value would make the most sense since the address is unlikely to be changed that much. (disregard comment, it's copied from someplace else and I'll delete it).

Reply:
You can have a default value with pflags too!

Reply:
But isn't it more logical to make something configurable in helm rather than in the code ?